partiql-spi/src/main/java/org/partiql/spi/Context.java

```diff
@@ -0,0 +1,44 @@
+package org.partiql.spi;
+
+import org.jetbrains.annotations.NotNull;
+import org.partiql.spi.errors.PErrorListener;
+
+/**
+ * Represents a common set of arguments across multiple major PartiQL components.
+ * @see Context#standard()
+ * @see Context#of(PErrorListener)
+ */
+public interface Context {
+
+    /**
+     * The default utilizes {@link PErrorListener#abortOnError()}.
+     * @return the registered {@link PErrorListener}.
+     */
+    @NotNull
+    default PErrorListener getErrorListener() {
+        return PErrorListener.abortOnError();
+    }
```

Comment:
I'm not sure we want to instantiate a new error listener every time, this is somewhat like the argument changing everytime. We can have a TODO to make AbortOnError a static variable